tests/view/downcastwriter/mergecontainers.js

```diff
@@ -8,8 +8,15 @@ import { stringify, parse } from '../../../src/dev-utils/view';
 
 import Document from '../../../src/view/document';
 import { expectToThrowCKEditorError } from '@ckeditor/ckeditor5-utils/tests/_utils/utils';
+import { StylesProcessor } from '../../../src/view/stylesmap';
 
 describe( 'DowncastWriter', () => {
+	let stylesProcessor;
+
+	beforeEach( () => {
+		stylesProcessor = new StylesProcessor();
```

Comment:
This should be inlined. Even though the test uses `before()` to created downcast writer :/